usImage.cpp

```diff
@@ -221,8 +221,7 @@ static unsigned char *buildGammaLookupTable(int blevel, int wlevel, double power
     if (blevel > 0xffff) blevel = 0xffff;
     if (wlevel > 0xffff) blevel = 0xffff;
 
-    for (int i = 0; i <= blevel; ++i)
-        result[i] = 0;
+    memset(result, 0, blevel + 1);

```

Comment:
I suggest reverting these memsets. They introduce some fragility in implicitly assuming sizeof(*result) is 1. I could be wrong but this feels more like a personal preference of using memset over an explicit loop. Did you really see any measurable difference from this change?

Reply:
It's a micro optimization, I agree to revert it.